iot/api-client/manager/README.md

```diff
@@ -134,3 +134,80 @@ Patch a device with RSA:
         -project_id=blue-jet-123 -pubsub_topic=hello-java \
         -registry_name=hello-java -rsa_certificate_file ../rsa_cert.pem \
         -device_id="java-device-0" -command=patch-device-rsa
+
+
+# Cloud IoT Core Java HTTP example
+
+This sample app publishes data to Cloud Pub/Sub using the HTTP bridge provided
+as part of Google Cloud IoT Core.
+
+Note that before you can run the sample, you must configure a Google Cloud
+PubSub topic for Cloud IoT Core and register a device as described in the
+[parent README](../README.md).
+
+## Setup
+
+Run the following command to install the dependencies using Maven:
+
+    mvn clean compile
+
+## Running the sample
+
+The following command summarizes the sample usage:
+
+```
+    mvn exec:java \
+        -Dexec.mainClass="com.example.cloud.iot.examples.HttpExample" \
```

Comment:
This seems long winded, but Ok

Reply:
Yeah, for the Manager sample, there are a few things that can are put in environment variables but for the device samples it would pretty much just be the project ID, which doesn't help much with the command length.